Add atomic-deployment flag (#757)
kyma-project/cli: docs/gen-docs/kyma_alpha_deploy.md

## Options

```bash

>   -a, --atomic                    Use atomic deployment (rollback any component which could not be successfully installed). (default true)

Reviewer: ```suggestion
  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully (default: true).
```

>       --cancel-timeout duration   Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client. (default 15m0s)

Reviewer: Does the timeout affect several workers or just one worker? If just one, use "worker's" instead of "worker'"

```suggestion
      --cancel-timeout duration   Time after which the workers' context is canceled (default: 15m0s). Any pending worker goroutines that are blocked by a Helm client will continue.
```

Author: the timeout affects all workers

>   -c, --components string         Path to the components file. (default "workspace/installation/resources/components.yaml")

Reviewer: ```suggestion
  -c, --components string         Path to the components file (default: "workspace/installation/resources/components.yaml")
```

Author: Good input, but we need to handled that decoupled form this PR as it requires a change in the doc-generator logic.

>   -d, --domain string             Domain used for installation. (default "local.kyma.dev")

Reviewer: ```suggestion
  -d, --domain string             Domain used for installation (default: "local.kyma.dev")
```
